Application/EdFi.Ods.Api/Security/AuthorizationStrategies/Relationships/RelationshipsAuthorizationStrategyFilterDefinitionsFactory.cs

```diff
@@ -94,6 +94,12 @@ protected IEnumerable<ViewBasedAuthorizationFilterDefinition> CreateAllEducation
             if (filterContext.SubjectEndpointValue == null)
             {
                 // This should never happen
```

Comment:
@semalaiappan 
This comment "// This should never happen" seems inapplicable since this scenario should happen.

Reply:
removed